sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/core/CosmosTemplateIT.java

```diff
@@ -228,6 +238,38 @@ public void testInsertDuplicateIdShouldFailWithConflictException() {
         }
     }
 
+    @Test
+    public void testInsertDocShouldNotPersistTransientFields() {
+        final Person personWithTransientField = TEST_PERSON_4;
+        assertThat(personWithTransientField.getTransientProperty()).isNotNull();
```

Comment:
Instead of isNotNull() can we just check that this equals Test constants.TRANSIENT_PROPERTY?